src/logic/addons.h

```diff
@@ -66,6 +66,7 @@ bool is_newer_version(const AddOnVersion& base, const AddOnVersion& compare);
 
 // Required add-ons for an add-on, map, or savegame with the recommended version
 using AddOnRequirements = std::vector<std::pair<std::string, AddOnVersion>>;
+using AddOnConflict = std::pair<std::string, bool>;
```

Comment:
This needs some documentation what the two parts mean, e.g.
```suggestion
using AddOnConflict = std::pair<std::string /* localized message */, bool /* critical */>;
```